src/main/java/matching_algorithm_use_case/MatchingAlgorithmResponseModel.java

```diff
@@ -0,0 +1,21 @@
+package matching_algorithm_use_case;
+
+import java.util.List;
+
+public class MatchingAlgorithmResponseModel {
+    String updateMessage;
+    List<String> groupToStringList;
+
+    public MatchingAlgorithmResponseModel(String uM, List<String> g){
+        this.updateMessage = uM;
```

Comment:
Just a general note, I think you should write your input variables as like updateMatches to follow convention and make it easier for whoever is reading.